src/resources/postcss/views/month/_multiday-event.pcss

```diff
@@ -26,6 +50,15 @@
 		}
 	}
 
+	.tribe-events-calendar-month__multiday-wrapper {
+
+		@media (--viewport-medium) {
```

Comment:
shouldn't need this

Reply:
Yes, it is required, because if not when switching to mobile, it continue taking the height and it looks weird: https://cloudup.com/cT2HSGJN2nL

Reply:
So in https://github.com/moderntribe/the-events-calendar/pull/2537/files#diff-a7442de4a8d47551d117e298780f4c19R30 I'll be adding separate markup for mobile and hiding the mobile on desktop (just the way mobile works, we can't use the same markup). `.tribe-events-calendar-month__day-cell` should be hidden on mobile and display on desktop.

If you want, you can merge as is and I'll fix it up on my branch when I merge the release branch into the mobile work.

Reply:
Okies.